quickstarts/karaf-features/src/main/resources/features.xml

```diff
@@ -118,5 +118,11 @@
         <feature version="${karaf.version.org.kie}">kie-spring</feature>
         <bundle>mvn:org.jboss.integration.fuse.quickstarts/spring-jbpm-osgi-example/${project.version}</bundle>
     </feature>
+    <feature name="fuse-bxms-switchyard.drools.channels" version="${project.version}">
```

Comment:
I think it would be better to use "-" instead of ".", just to be consistent with the other feature names.